packages/design-system/src/components/N8nSticky/__tests__/Sticky.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { render, fireEvent } from '@testing-library/vue';
+import N8nSticky from '../Sticky.vue';
+import { N8nMarkdown } from '@/main';
```

Comment:
To avoid loading the entire package in this test:

```suggestion
import N8nMarkdown from '../../N8nMarkdown/Markdown.vue';
```